packages/editor/src/components/index.js

```diff
@@ -8,6 +8,7 @@ export { default as BlockEdit } from './block-edit';
 export { default as BlockFormatControls } from './block-format-controls';
 export { default as BlockNavigationDropdown } from './block-navigation/dropdown';
 export { default as BlockIcon } from './block-icon';
+export { default as BlockVerticalAlignmentToolbar } from './block-vertical-alignment-toolbar';
```

Comment:
It would be essential to include docs for this new component given that is going to be exposed as part of the public API. Blocks developers will need to see some code examples to know how to use it in their projects.

Reply:
Great point. I did consider this but there didn't seem to be a standard "docs" template to use. If there is one I'm happy to spin up some quick docs. If not a recommended example to follow would be very much appreciated.

Reply:
Ok I've addressed this in https://github.com/WordPress/gutenberg/pull/13899/commits/ce39fabeb24a7ef4201116fa2dac623e5b1148a7

Let me know if that works.